src/routes/event-team-matches.tsx

```diff
@@ -25,8 +25,12 @@ const EventTeamMatches = ({ eventKey, teamNum }: Props) => {
   const matches = useEventMatches(eventKey, 'frc' + teamNum)
   return (
     <Page
-      name={`Matches: ${teamNum} @ ${eventName}`}
-      back={`/events/${eventKey}/teams/${teamNum}`}
+      name={
+        <>
+          {`Matches: ${teamNum} @ `}
+          <a href={`/events/${eventKey}`}> {eventName} </a>
```

Comment:
We do not need these extra spaces
```suggestion
          <a href={`/events/${eventKey}`}>{eventName}</a>
```